microsoft-365/security/defender/investigate-dlp.md

```diff
@@ -86,6 +86,9 @@ Before you start, [turn on alerts for all your DLP policies](/microsoft-365/comp
 
 10. Go to the incident summary page and select **Manage Incident** to add incident tags, assign, or resolve an incident.
 
+> [!IMPORTANT] 
+> DLP supports associating DLP policies and alert management with administrative units in the Microsoft Purview compliance portal (preview). DLP alerts are only available to unrestricted DLP administrators in the Microsoft 365 Defender portal. Administrative unit restricted DLP administrator will not see DLP alerts.  See [Administrative units](/microsoft-365/compliance/microsoft-365-compliance-center-permissions.md#administrative-units-preview) for implementation details. See, [policy scoping](/microsoft-365/compliance/dlp-policy-reference.md#policy-scoping) for details on administrative unit scoping. 
```

Comment:
@chrfox for statement "Admin Unit restricted DLP administrator will not see DLP alerts" should we mention until Admin units are supported in DLP or will we just update the doc when that is available in preview?